backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -358,9 +345,8 @@ export class UserResolver {
       await queryRunner.commitTransaction()
       logger.addContext('user', dbUser.id)
     } catch (e) {
-      logger.error(`error during create user with ${e}`)
       await queryRunner.rollbackTransaction()
-      throw e
+      throw new LogError(`Error during create user with error: ${e}`, e)
```

Comment:
May be we expose too much information here?
```suggestion
      throw new LogError('Error during create user', e)
```

Reply:
I decided for `throw new LogError('Error creating user', e)` since thats better English